docker/tanner/template injection/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine
+
+RUN apk -U --no-cache add \
+  python3
+
+RUN pip3 install --upgrade pip && \
+    pip3 install --no-cache-dir \
+    tornado \
+    jinja2 \
+    mako
+
+CMD ["python3", "-c", "import tornado"]
```

Comment:
Is it required?

Reply:
It was to test if everything works perfectly or not.